packages/core-data/src/actions.js

```diff
@@ -94,6 +94,22 @@ export function receiveEntityRecords(
 	};
 }
 
+export function deleteEntityRecords(
+	kind,
+	name,
+	records,
+	query,
+	invalidateCache = true
+) {
+	return {
+		type: 'DELETE_ITEMS',
+		items: castArray( records ),
+		kind,
+		name,
+		invalidateCache,
+	};
+}
```

Comment:
This action should be in `packages/core-data/src/queried-data/actions.js` which is where `receiveItems` is. 

I also think we should name it `removeItems` / `REMOVE_ITEMS` as the action removes data from the store and doesn't perform the actual `DELETE` request.

And, as I mentioned in another thread, we only need the `itemIDs`, not all of `items`.

Reply:
Agree on itemIds. On the location I figured we're not using any query, and I am not sure what is in querried-data and what is not in querried-data :)